src/components/Markdown/Headings.tsx

```diff
@@ -41,16 +39,18 @@ export const Headings: FunctionComponent<HeadingResolverProps> = ({
 
   const id = dataElement?.dataset.headingId ?? sanitize(title);
 
-  const isH3OrLarger = level < 4;
-
   return (
     <>
       <Heading
         as={elem}
-        color="blue.800"
+        backgroundColor={level >= 4 ? "gray.50" : undefined}
+        borderBottom="1px solid #eee"
+        color="gray.800"
         level={level}
-        mb={isH3OrLarger ? 0 : level}
-        mt={marginY}
+        marginBottom="0em"
```

Comment:
nit since I've noticed it in another spot, but you can just use `0` instead of specifying a unit